invenio_notify/constants.py

```diff
@@ -9,10 +9,24 @@
 SUPPORTED_TYPES = [TYPE_REVIEW, TYPE_ENDORSEMENT, TYPE_TENTATIVE_ACCEPT, TYPE_REJECT, TYPE_TENTATIVE_REJECT]
 """ List of supported notification types that can be processed by the notify  """
 
-# Status constants
-STATUS_REQUEST_ENDORSEMENT = 'Request Endorsement'
-
+# Workflow status constants
+WORKFLOW_STATUS_REQUEST_ENDORSEMENT = 'request_endorsement'
+WORKFLOW_STATUS_TENTATIVE_ACCEPT = 'tentative_accept'
+WORKFLOW_STATUS_TENTATIVE_REJECT = 'tentative_reject'
+WORKFLOW_STATUS_ANNOUNCE_REVIEW = 'announce_review'
+WORKFLOW_STATUS_ANNOUNCE_ENDORSEMENT = 'announce_endorsement'
+WORKFLOW_STATUS_REJECT = 'reject'
+WORKFLOW_STATUS_AVAILABLE = 'available' # This is not COAR standard, used internally.
 
+# Mapping from (activity, notification_type) to workflow status
+NOTIFICATION_TO_WORKFLOW_STATUS = {
+    ('Offer', TYPE_ENDORSEMENT): WORKFLOW_STATUS_REQUEST_ENDORSEMENT,
```

Comment:
we support incoming endorsement request
better remove this or 
filter it in inbox_processing

Reply:
This isn't meant for incoming endorsement requests but to track state of workflow 

`create_endorsement_request_record` will now set initial state to this status.

Reply:
You're right that we don't support incoming requests, but my idea wasn't to do so but to use one set of states that could be shared between unsolicited reviews/endorsements as well as the PCI workflow.

This was also partly tied into the db queries done for `has_available_actors` so that the returned statuses would not be different between the two scenarios.

Like I say, in the PR `create_endorsement_request_record`  now sets initial status of outgoing request to that value, but if it's not good idea to include a subquery to the endorsement request table, then this all seems moot.

Reply:
Thanks @philipkcl, you're right. I'm not thinking clearly about this.